docs/operation/cluster_and_service_configuration.rst

```diff
@@ -610,7 +610,7 @@ To change the number of replicas, perform the following operations:
 
       root\@bootstrap $ kubectl exec --kubeconfig /etc/kubernetes/admin.conf \\
                          -n kube-system -c salt-master salt-master-bootstrap \\
-                         -- salt-run state.sls metalk8s.deployed \\
+                         -- salt-run state.sls metalk8s.logging.deployed \\
```

Comment:
The `Replicas Count Customization` is not only about Loki, but for all services using the CSC mechanism.

Reply:
Ah right, went too fast. Good catch :ok_hand:

Reply:
Done